src/Course/Applicative.hs

```diff
@@ -37,53 +37,54 @@ infixl 4 <*>
 
  prop> \x -> pure x == ExactlyOne x
 
- >>> ExactlyOne (+10) <*> ExactlyOne 8
- ExactlyOne 18
+-- > pure (+10) <*> MakeExactlyOne 8
+ MakeExactlyOne 18
 -}
 instance Applicative ExactlyOne where
     pure :: a -> ExactlyOne a
-    pure = error "todo: Course.Applicative pure#instance ExactlyOne"
+    pure = MakeExactlyOne
 
     (<*>) :: ExactlyOne (a -> b) -> ExactlyOne a -> ExactlyOne b
-    (<*>) = error "todo: Course.Applicative (<*>)#instance ExactlyOne"
+    (<*>) (MakeExactlyOne f) = mapExactlyOne f
```

Comment:
Excellent! I like how you use `mapExactlyOne` instead of explicitly matching the argument.